director/api/public.py

```diff
@@ -1,9 +1,40 @@
-from flask import Blueprint
-from director.model import User
-
+from flask import Blueprint, Response,  request, make_response
+from director.model import User, Terminal
+from director.authentication import  token_required , generate_Jwt, Authenticator
+secret = 'pass'
+valid_password = 'pass'
 PublicAPI = Blueprint('public_api', __name__, url_prefix='/api/public')
 
+
+"""
+public api login with ldap authentication and returns a JWT
+"""
+@PublicAPI.route('/login')
+def login():
+    auth = request.authorization
```

Comment:
`authorization` is an interesting idea here, but don't you think using a POST field username + password combination would serve us better here?